scanner/src/main/kotlin/Scanner.kt

```diff
@@ -56,6 +56,7 @@ import org.ossreviewtoolkit.model.utils.FileArchiver
 import org.ossreviewtoolkit.model.utils.ProvenanceFileStorage
 import org.ossreviewtoolkit.model.utils.getKnownProvenancesWithoutVcsPath
 import org.ossreviewtoolkit.model.utils.vcsPath
+import org.ossreviewtoolkit.scanner.FileList as ScannerFileList
```

Comment:
As an unrelated side note, I was even aware (or forgot about it) that we do have both

    org.ossreviewtoolkit.model.FileList
    org.ossreviewtoolkit.scanner.FileList

which seem to be awfully similar. @fviernau do you recall why we have both of them?

Reply:
Yes, the uses are decoupled `OrtResult` vs. `ProvenanceFileStorage` and others.

Reply:
So, in order to reduce the confusion about the usage a bit, what do you think about actually renaming `org.ossreviewtoolkit.scanner.FileList` to `ScannerFileList` to avoid the need of an alias-import as done here?

Reply:
To me it is not confusion that different layers may use similar terms for a different thing. 
I believe it's fine to have the same name in different modules.